Jenkinsfile

```diff
@@ -0,0 +1,89 @@
+def untrusted = false
+
+pipeline {
+  agent none
+  options {
+    timeout(time: 10, unit: 'HOURS')
+    checkoutToSubdirectory('sssd')
+  }
+  environment {
+    NAME       = "$BRANCH_NAME/$BUILD_ID"
+    BASE_DIR   = "/home/fedora"
+    GH_CONTEXT = "sssd-ci"
+    GH_SUCCESS = "Success."
+    GH_PENDING = "Build is pending."
+    GH_FAILURE = "Build failed."
+    GH_URL     = "https://sssd-jenkins.duckdns.org/blue/organizations/jenkins/sssd-ci/detail/$BRANCH_NAME/$BUILD_ID/pipeline"
```

Comment:
Is this where the artifacts are being uploaded to? 

It's not clear to me what the relationship is between this and the AWS base as later the script later rewrites this variable.

btw the certificate seems incorrect, at least my firefox chokes on it :)

Reply:
This is url into internal jenkins instance. Normally available on internal dns name the we should not make public though. That's why certificate is incorrect.

Reply:
You can go to internal jenkins for this PR by clicking on details next to sssd-ci and you can see public logs in aws with sssd-ci/fedora28.

Reply:
This is just a shortcut to the task, it is not necessary to publish it. In that case we could just remove the 'sssd-ci' status report. You can access the jenkins through internal dns name, the certificate will be valid there.